Sources/AudioKit/MIDI/MIDIInstrument.swift

```diff
@@ -38,8 +38,8 @@ open class MIDIInstrument: PolyphonicNode, MIDIListener, NamedNode {
     ///   - name: Name to connect with
     ///
     open func enableMIDI(_ midiClient: MIDIClientRef = MIDI.sharedInstance.client,
-                         name: String = "AudioKit MIDI Instrument") {
-        CheckError(MIDIDestinationCreateWithBlock(midiClient, name as CFString, &midiIn) { packetList, _ in
+                         name: String? = "AudioKit MIDI Instrument") {
```

Comment:
Here again, it seems very strange and undesirable to making `name` optional. A default is already provided, so if no value is passed in, the default is used. This would mean the only way to get `nil` here would be for the caller to explicitly pass in `name: nil`.

Reply:
I agree lets avoid optionals.